src/Drupal/DrupalExtension/Context/RawDrupalContext.php

```diff
@@ -160,6 +160,27 @@ public function getRandom() {
   }
 
   /**
+   * Massage node values to match the expectations on different Drupal versions.
+   *
+   * @beforeNodeCreate
+   */
+  public function alterNodeParameters(BeforeNodeCreateScope $scope) {
+    $drupal_version = $scope->getContext()->getDrupal()->getDriver()->version;
+    $node = $scope->getEntity();
+
+    // On Drupal 8 the timestamps should be in UNIX time.
+    switch ($drupal_version) {
+      case 8:
+        foreach (array('changed', 'created', 'revision_timestamp') as $field) {
+          if (!empty($node->$field) && !is_numeric($node->$field)) {
+            $node->$field = strtotime($node->$field);
+          }
+        }
+      break;
+    }
+  }
```

Comment:
When saving nodes, the timestamps should be in UNIX time on Drupal 8, while Drupal 7 accepts a string. I've solved it here in a `@beforeNodeCreate` hook. Is this OK, or would it be better to handle this in DrupalDriver?